api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java

```diff
@@ -540,7 +540,7 @@ public static String getPatientsWithObsBetweenDates(
     s.append("WHERE e.location_id = :location AND e.encounter_type = ${encounterType} ");
```

Comment:
@iteliolucas can you please change this names for sugestive names like like the proper name of encounter something like this, is difficult to see if this query is ohk or not like this 
```suggestion
    s.append("WHERE e.location_id = :location AND e.encounter_type = ${adultoSeguimentoEncounterType} ");
```

Reply:
I guess this is like this cause is generic one, never mind the sugestion.....
Please move this to genericQueries